pmd-core/src/test/java/net/sourceforge/pmd/ant/PMDTaskTest.java

```diff
@@ -78,7 +78,7 @@ public void testWithShortFilenames() throws FileNotFoundException, IOException {
             String actual = IOUtils.toString(in, StandardCharsets.UTF_8);
             // remove any trailing newline
             actual = actual.replaceAll("\n|\r", "");
-            Assert.assertEquals("sample.dummy:0:\tTest Rule 2", actual);
+            Assert.assertEquals("sample.dummy:0:\tTest Rule 2:\tSampleXPathRule", actual);
```

Comment:
I think it would be better to place the rule name before the message. Messages are of varying lengths and the rule names will end up misaligned

Reply:
Hi, 
Ok , no problem. I put it there for the moment because @Fernal73 pointed out that this could popetntially break backwards compatibility (see comment https://github.com/Fernal73 )

Reply:
Changed the output to put the RuleName before the description.

Reply:
Fine with me if backwards compatibility is not an issue. As long as nobody is parsing and stuffing this into a database. Would anybody do this for the text output, anyway?

Reply:
> Would anybody do this for the text output, anyway?

Exactly, I wouldn't be concerned too much about backwards compatibility for this text format. Mentioning the change in the release notes should be enough

Thanks for making the change